gobblin-metastore/build.gradle

```diff
@@ -77,6 +77,9 @@ artifacts {
 
 test {
     workingDir rootProject.rootDir
+    if (project.hasProperty('usePreinstalledMysql') && Boolean.parseBoolean(project.usePreinstalledMysql)) {
+      jvmArgs '-Dgobblin.metastore.testing.embeddedMysqlEnabled=false'
```

Comment:
how are pre installed credentials picked up? are the hardcoded in the tests and expected to be the same? what if the state (test tables, etc) are not cleaned up from pre-installed mysql if tests fail / jvm crashes?